packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3853,42 +3853,41 @@ curl -v -X POST \
 
 ```json
 {
-  "id": "smsk33ujQ59REImFX0g3",
-  "factorType": "sms",
-  "provider": "OKTA",
-  "status": "ACTIVE",
-  "created": "2015-02-04T07:07:25.000Z",
-  "lastUpdated": "2015-02-04T07:07:25.000Z",
-  "profile": {
-    "phoneNumber": "+1415551337"
-  },
-  "_links": {
-    "verify": {
-      "href": "https://{yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors/smsk33ujQ59REImFX0g3/verify",
-      "hints": {
-        "allow": [
-          "POST"
-        ]
-      }
-    },
-    "self": {
-      "href": "https://{yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors/smsk33ujQ59REImFX0g3",
-      "hints": {
-        "allow": [
-          "GET",
-          "DELETE"
-        ]
-      }
+    "id": "emfnf3gSScB8xXoXK0g3",
+    "factorType": "email",
```

Comment:
What was the motivation for changing the factor from SMS to Email?

Reply:
@edjohns-okta rather than customize the existing object by adding the parameter `vendorName`, I thought it best to copy in a complete object, so, took one from one of the other examples. Would it be preferable to add the factor email here?

Reply:
@brianduffield-okta Gave you a ship.  I'm assuming the Vendor property is found on all authenticators.  If not, you will need to appropriately modify this doc